mws/mws.go

```diff
@@ -72,7 +72,7 @@ type Network struct {
 	GcpNetworkInfo   *GcpNetworkInfo      `json:"gcp_network_info,omitempty"`
 }
 
-// GcpVpcEndpointInfo is the objecy that configures GCP Private Service Connect endpoints.
+// GcpVpcEndpointInfo is the objecy that configures GCP Private Service Connect serving.
```

Comment:
is it really `serving`, not `endpoints`?

Reply:
pitfall of bulk replacing